aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -148,21 +150,45 @@ Tensor& copysign_(Tensor& self, Scalar other) {
   return native::copysign_(self, wrapped_scalar_tensor(other));
 }
 
-Tensor& div_out(const Tensor& self, const Tensor& other, Tensor& result) {
+Tensor& div_true_out(const Tensor& self, const Tensor& other, Tensor& result) {
   auto iter = TensorIterator::binary_float_op(result, self, other);
-  div_stub(iter.device_type(), iter);
+  div_true_stub(iter.device_type(), iter);
+  if (!result.defined()) {
```

Comment:
Do the internals of this function need to change?

Reply:
Doing it this way means I can write `div(self, other)` as
```cpp
Tensor result;
div_out(self, other, result);
return result;
```

Which means I don't have to duplicate the string comparison `if-else` chain.

Reply:
OK, sounds good.